.markdownlint-cli2.jsonc

```diff
@@ -28,6 +28,7 @@
         "EXCLUDE",
         "Execution",
         "EXTERNAL",
+        "Fabric",
```

Comment:
I'll probably just remove this. I thought it was cool, but it's probably too much maintenance for little gain